src/ci/docker/host-x86_64/sbf-solana-solana/Dockerfile

```diff
@@ -36,3 +36,9 @@ ENV SCRIPT CARGO_TARGET_BPFEL_UNKNOWN_UNKNOWN_RUNNER=\"cargo-run-bpf-tests --hea
     PATH="${HOME}/.cargo/bin:${PATH}" \
     python3 /checkout/x.py --stage 1 test --host='' --target bpfel-unknown-unknown \
     library/core
+
+ENV SCRIPT CARGO_TARGET_SBF_SOLANA_SOLANA_RUNNER=\"cargo-run-bpf-tests --heap-size 104857600\" \
+    LLVM_HOME=/checkout/obj/build/x86_64-unknown-linux-gnu/llvm \
+    PATH="${HOME}/.cargo/bin:${PATH}" \
+    python3 /checkout/x.py --stage 1 test --host='' --target sbf-solana-solana \
```

Comment:
I think we should make it `--target sbf-solana-solana,bpfel-unknown-unknown` so we test both. We also need to set  `CARGO_TARGET_BPFEL_UNKNOWN_UNKNOWN_RUNNER` above.